flask_monitoringdashboard/controllers/requests.py

```diff
@@ -1,13 +1,14 @@
 import datetime
-
 import numpy
-from sqlalchemy import func, and_
-
 from flask_monitoringdashboard.core.timezone import to_utc_datetime, to_local_datetime
-from flask_monitoringdashboard.database import Request
 from flask_monitoringdashboard.database.count_group import count_requests_per_day, get_value
 from flask_monitoringdashboard.database.endpoint import get_endpoints, get_num_requests
-from flask_monitoringdashboard.database.request import create_time_based_sample_criterion
+from flask_monitoringdashboard.database.request import \
+    get_status_code_frequencies, get_all_request_status_code_counts, get_error_requests_db
```

Comment:
if it's not too much work, use the following syntax for importing many functions:
```
from .... import (
    function_a,
    function_b,
)
```

Reply:
Done